xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/docextra.vm

```diff
@@ -91,23 +91,24 @@
 #if($docextras.size() > 0)
   <div id="xwikidatacontents">
     <div class="floatcontainer" id="docExtraTabs">
-      <ul class="xwikitabbar" id="docExtrasTabsUl">
+      <div id='tablistlabel' class='hidden'>$escapetool.xml($services.localization.render('docextra.tablist.label'))</div>
```

Comment:
Why are you using a `hidden` class here and not just the [hidden attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden)?

Reply:
This class was already used extensively in this file, notably on https://github.com/xwiki/xwiki-platform/blob/e7b6d35fbb7dd55af958496a0e6c19e344411817/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/docextra.vm#L111 
that was written in 2014.

___
In regards to the discussion about it on https://github.com/xwiki/xwiki-platform/pull/2182#discussion_r1268224725 , I decided to change this use of "hidden" to the `sr-only` class. Addressed in https://github.com/xwiki/xwiki-platform/pull/2165/commits/19b1cf391baa5ea4920bd157ee146f138d3e550c